examples/angular-cli/src/stories/__snapshots__/addon-actions.stories.storyshot

```diff
@@ -1,37 +1,47 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Storyshots Addon/Actions Action and method 1`] = `
-<storybook-dynamic-app-root
-  cfr={[Function CodegenComponentFactoryResolver]}
-  data={[Function Object]}
-  target={[Function ViewContainerRef_]}
+<storybook-wrapper
```

Comment:
Good catch @gaetanmaisse on the snapshots. Is it easy to make this a non-breaking change?